dissect/target/target.py

```diff
@@ -481,6 +482,14 @@ def _init_os(self) -> None:
         self._os_plugin = os_plugin
         self._os = self.add_plugin(os_plugin.create(self, fs))
 
+    def _mount_others(self) -> None:
+        root_fs = self.fs
+        index = 0
+        for fs in self.filesystems:
+            if fs not in root_fs.mounts.values():
+                root_fs.mount(f"/$fs$/fs{index}", fs)
```

Comment:
Can you add a check for if `$fs$` already exists?

Reply:
Or maybe under a different index? So you can re-run this function if you've added new filesystems.